x-pack/plugins/infra/common/log_sources/resolved_log_source_configuration.ts

```diff
@@ -109,6 +109,7 @@ const resolveRuntimeMappings = (indexPattern: IndexPattern): estypes.MappingRunt
   const runtimeMappingsFromIndexPattern = (Object.entries(runtimeFields) as ObjectEntries<
     typeof runtimeFields
   >).reduce<estypes.MappingRuntimeFields>(
+    // @ts-expect-error @elasticsearch/elasticsearch does not support yet "composite" type for runtime fields
```

Comment:
This causes a

```
x-pack/plugins/infra/common/log_sources/resolved_log_source_configuration.ts:130:3 - error TS2578: Unused '@ts-expect-error' directive.
```

during bootstrap, even though it is clearly required. Is that a known issue? How will we know when it's actually not required anymore?

Reply:
It's a good question and I can't tell you how we could be pinged when the underlying `@elasticsearch/elasticsearch` is updated with the new supported type.  
Any idea @elastic/kibana-operations ?